lls/main.c

```diff
@@ -319,13 +385,25 @@ lls_proc(void *arg)
 
 	while (likely(!exiting)) {
 		/* Read in packets on front and back interfaces. */
-		process_pkts(lls_conf, &net_conf->front,
+		uint16_t num_tx = process_pkts(lls_conf, &net_conf->front,
 			lls_conf->rx_queue_front, lls_conf->tx_queue_front);
 		if (net_conf->back_iface_enabled)
-			process_pkts(lls_conf, &net_conf->back,
+			num_tx += process_pkts(lls_conf, &net_conf->back,
 				lls_conf->rx_queue_back,
 				lls_conf->tx_queue_back);
 
+		if (num_tx > 0 && (lacp_enabled(net_conf, &net_conf->front) ||
+				lacp_enabled(net_conf, &net_conf->back))) {
+			int ret = rte_timer_reset(&lls_conf->lacp_timer,
```

Comment:
The call of `rte_timer_reset()` on `lls_conf->lacp_timer` is quite special, i.e., many specific parameters. It'd be better to encapsulate it into an inline function and replace all current calls with the new inline function.